wled00/bus_manager.h

```diff
@@ -150,6 +155,32 @@ class Bus {
         if (_cctBlend > WLED_MAX_CCT_BLEND) _cctBlend = WLED_MAX_CCT_BLEND;
       #endif
     }
+    static void calculateCCT(uint32_t c, uint8_t &ww, uint8_t &cw) {
```

Comment:
This can be a private or protected, does not need to be public method.

Reply:
Done